SDK/bundles/org.eclipse.embedcdt.debug.gdbjtag.qemu.core/src/org/eclipse/embedcdt/debug/gdbjtag/qemu/core/dsf/DebuggerCommands.java

```diff
@@ -190,5 +190,11 @@ public IStatus addStartRestartCommands(boolean doReset, List<String> commandsLis
 		return Status.OK_STATUS;
 	}
 
+	@Override
+	public IStatus addRegisterFileCommands(List<String> commandsList, String regFilePath) {
+		// TODO Auto-generated method stub
+		return null;
```

Comment:
Why this is returning null?

Reply:
In qemu we don't have target description feature, changes are done